app/index.html

```diff
@@ -53,6 +53,7 @@
 
         <!-- build:js js/app.min.js -->
         <script src="components/socket.io-client/socket.io.js"></script>
+        <script src="components/jquery/dist/jquery.js"></script>
```

Comment:
Do we really need to use jquery for this :cry: ? I know `ngInfiniteScroll` depends on this but it seems a bit over the top to have to include this library just for this module. Maybe we should explore alternatives or remove infinite scrolling and use regular pagination instead.